examples/braket_features/Verbatim_Compilation.ipynb

```diff
@@ -792,6 +792,132 @@
     "print(result.measurement_counts)"
```

Comment:
<p>The MS gate looks weird; we should fix that in the SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>"looks weird" lol it's incorrect. Extra paren is already removed though, so this notebook just needs to be reran to have output adjusted</p>